lib/erb_lint/cache.rb

```diff
@@ -9,6 +9,8 @@ def initialize(config, cache_dir = nil)
       @cache_dir = cache_dir || CACHE_DIRECTORY
       @hits = []
       @new_results = []
+      @gemfile_lock = File.read("Gemfile.lock") if File.exist?("Gemfile.lock")
+      @rubocop_config = File.read(".rubocop.yml") if File.exist?(".rubocop.yml")
```

Comment:
Additionally, if there's a way to fetch this from `linters.Rubocop.rubocop_config.inherit_from`, that seems worthwhile, since it is parameterized in the first place: https://github.com/goatapp/rubocop/blob/97e4ffc8a71e9e5239a927c6a534dfc1e0da917f/manual/configuration.md?plain=1#L81

Also, I don't know how common it is, but it looks like this library does allow for URIs there also: https://github.com/goatapp/rubocop/blob/97e4ffc8a71e9e5239a927c6a534dfc1e0da917f/manual/configuration.md?plain=1#L86

Actually, given the complexity / variability there, if there's a way to access an instance of the Rubocop linter here, it may be worth removing dependence on the source of that configuration.

Reply:
Also, pardon the commentary. I think this is a good PR, and I don't want to let solving for all the cases get in the way of anything; just thinking reading through the code and thinking aloud.

It looks like runner context is available in `run_using_cache`, so maybe some "runner checksum" could be passed an optional arg to `cache.get()` and `cache.set()`? That solves for:
- through the runner, putting linters in charge of their own checksum logic
- not being coupled to / duplicating awareness of how linters are configured

I'll give this some thought over the weekend and see if I come up with anything useful.